docs/concepts/service-catalog/index.md

```diff
@@ -20,10 +20,10 @@ Using Service Catalog, a {% glossary_tooltip text="Cluster Operator" term_id="cl
 
 An {% glossary_tooltip text="Application Developer" term_id="application-developer" %} wants to use message queuing as part of their application running in a Kubernetes cluster.
 However, they do not want to deal with the overhead of setting such a service up and administering it themselves.
-Fortunately, there is a cloud provider that offers message queuing as a *Managed Service* through their *Service Broker*.
+Fortunately, there is a cloud provider that offers message queuing as a Managed Service through their Service Broker.
 
-A {% glossary_tooltip text="Cluster Operator" term_id="cluster-operator" %} can setup Service Catalog and use it to communicate with the cloud provider's {% glossary_tooltip text="Service Broker" term_id="service-broker" %} to provision an instance of the message queuing service and make it available to the application within the Kubernetes cluster.
-The {% glossary_tooltip text="Application Developer" term_id="application-developer" %} therefore does not need to concern themselves with the implementation details or management of the message queue.
+A Cluster Operator can setup Service Catalog and use it to communicate with the cloud provider's Service Broker to provision an instance of the message queuing service and make it available to the application within the Kubernetes cluster.
+The Application Developer therefore does not need to concern themselves with the implementation details or management of the message queue.
 Their application can simply use it as a service.
 
 ## Architecture
```

Comment:
"It is implemented as an extension API server and a controller manager.

I think this should be "controller", not "controller manager".

https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/design.md

Reply:
done